README.md

```diff
@@ -1,63 +1,100 @@
 # Cartography
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?style=flat-square)](LICENSE)![Version](https://img.shields.io/nuget/v/Cartography.DynamicMap?style=flat-square)![Downloads](https://img.shields.io/nuget/dt/Cartography.DynamicMap?style=flat-square)   
 
-Nventive solution for mobile app map.
-
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+Cartography provides you 3 map related libraries for iOS, Android and Windows using native map of each platform.
 
 ## Getting Started
 
 ### **Samples**
 - Clone project a sample is available.
-- For seeing samples, build and install app with VS on the desire device (Android, IOS, or UWP)
+- For seeing samples, build and install app with VS on the desire device (Android, IOS, or UWP).
```

Comment:
Checking the samples should not be steps of the Getting Started section.
Please check the following link for the content and structure of the readme: https://dev.azure.com/nventive/Practice%20committees/_wiki/wikis/Committees.wiki/14813/Recipes?anchor=repository-structure-and-documentation